app/models/asset_field_type.rb

```diff
@@ -52,6 +53,29 @@ def mapping
 
   private
 
+  def is_image(mime_type)
+    ['image/jpeg', 'image/pjpeg', 'image/png','application/pdf' ,'image/x-png', 'image/gif'].include?(mime_type)
+  end
+
+  def format_data(metadata_hash)
+   return metadata_hash unless is_image(metadata_hash[:content_type])
+   metadata_hash.merge(image_asset_styles)
+  end
+
+  def image_asset_styles
+    {
+      styles: {
+        large: {geometry: '1800x1800>', format: :jpg},
+        medium: {geometry: '800x800>', format: :jpg},
+        default: {geometry: '300x300>', format: :jpg},
+        mini: {geometry: '100x100>', format: :jpg},
+        micro: {geometry: '50x50>', format: :jpg},
+        post_tile: {geometry: '1140x', format: :jpg}
+      },
+      processors: [:thumbnail, :paperclip_optimizer]
```

Comment:
Again, not every user that would touch this would necessarily want thumbnails with all image assets. As far as assumptions go it's not the worst we could make, but we still want to avoid assuming anything about future User's needs / want to leave everything as flexible and customizable as possible